src/shapes/Line.ts

```diff
@@ -13,31 +13,31 @@ const coordProps = ['x1', 'x2', 'y1', 'y2'];
 export class Line extends FabricObject {
```

Comment:
I am waiting for Line to die

Reply:
you will have to wait a bit longer, i wanted to remove them too, but we are too far now with other changes.
Also this is just  extra code and removing it is not giving us any major benefit if not re-documentin and killing existing stack overflow answers.
But yes one day those are just polygons, and probably polygons are just paths